charters/DataPortal/charter.md

```diff
@@ -0,0 +1,81 @@
+# Data Portal
+
+
+## Description
+The Data Portal provides the overall web-site for the HCA DCP with entry point from https://www.humancellatlas.org/. It provides the user experience for scientific and general users to upload, explore, search and run analyses on human genomic data. The Data Browser ("Explore") application provides a faceted search capability to find and download subsets of this data.
+
+## Objective
+Implementation of the Data Coordination Platform (DCP) website experience. Provide the access point for scientific and general users to search all HCA genetic data in order to identify a subset of the data for download and/or further analysis, providing file/bundle search on multiple clouds at PB scale, with strong public APIs.
+
+## In-scope
+### General
+* UX, visual and branding design with input from HCA leadership (GG, scientific steering committee, etc.)
```

Comment:
I would define the acronym `GG` as it will likely be unfamiliar to new visitors.

Reply:
The DSC - *scientific steering committee* dissolved last night.  @jonahcool - is the GG committed to offering input on _UX, visual and branding design_ ?  @theathorn - we're also calling out work items that require oversight from Science governance under _Scientific "guardrails" in the template. It flags items that may need more review by Science PMs. Is there a specific schedule or process in mind?